package.json

```diff
@@ -67,7 +67,7 @@
         "mailing-core": "^0.9.10",
         "micro": "^10.0.1",
         "neverthrow": "^6.0.0",
-        "next": "^13.4.9",
+        "next": "13.3.2",
```

Comment:
Is this the [issue](https://github.com/storybookjs/storybook/issues/22987) you were referring to?